octobot_trading/exchanges/types/websocket_exchange.py

```diff
@@ -309,9 +309,21 @@ def get_exchange_pair(self, pair: str) -> str:
         else:
             raise ValueError(f'{pair} is not supported on {self.get_name()}')
 
-    def safe_float(self, dictionary, key, default_value):
+    def safe_float(self, dictionary, key, default_value=None):
         return ccxtExchange.safe_float(dictionary, key, default_value)
 
+    def safe_integer(self, dictionary, key, default_value=None):
+        return ccxtExchange.safe_integer(dictionary, key, default_value)
+
+    def safe_string(self, dictionary, key, default_value=None, lower=False):
+        value = ccxtExchange.safe_string(dictionary, key, default_value)
+        if value is not None and lower:
+            value = value.lower()
+        return value
+
+    def iso8601(self, timestamp):
```

Comment:
I don't think it's a good idea to add those methods. Prefer using rest exchange parsers

Reply:
What do you mean ? When we can't parse elements from ccxt directly

Reply:
You can directly use the exchange client from the WS. I don't really understand why we need a wrapper here